service/controller/v3/resource/vnetpeering/error.go

```diff
@@ -0,0 +1,46 @@
+package vnetpeering
+
+import (
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/giantswarm/microerror"
+)
+
+var invalidConfigError = microerror.New("invalid config")
```

Comment:
Not sure if you missed the 100 PRs about rewriting errors but we want to do this differently now. See https://github.com/giantswarm/fmt/blob/master/go/errors.md.